### PR TITLE
fix: fixes path formatting issue in noRelativeImportCreate function

### PR DESCRIPTION
**Pull request by rinaldilucas**

### CODE REVIEW

File: lib/rules/no-relative-imports.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ function noRelativeImportCreate(context) {
       return;
     }
 
-    const expected = getAbsolutePathToTarget(path, tsconfigSettings);
+    let expected = getAbsolutePathToTarget(path, tsconfigSettings).replace(/\\/g, "/");
```
Comment (qdanik) [inline lib/rules/no-relative-imports.js:84]: I'm not sure that is the fix of your problem. 

For example method `getSlashCount`, `isMaxDepthExceeded` and `isRelativeToParent` also not working right.

Comment (qdanik) [reply lib/rules/no-relative-imports.js:84]: Okay, lets update `getSlashCounts`: 
```
function getSlashCounts(path) {
  if (!path) {
    return 0;
  }
  return (path.split(/[\\/]/).length || 1) - 1;
}
``` 
Then we should add one more method `replaceInvertedSplash` (you can place under `getSlashCounts`):
```
/**
 * Replace all backslashes (\) with forward slashes (/)  
 * @param {string} path
 * @returns {string}
 */
function replaceInvertedSplash(path) {
  if (!path) {
    return "";
  }
  return path.replace(/\\/g, "/");
}
```
Finally we need to wrap expected variable with new method:
```
const expected = replaceInvertedSplash(getAbsolutePathToTarget(path, tsconfigSettings));
```

Comment (rinaldilucas) [reply lib/rules/no-relative-imports.js]: That's awesome qdanik. If you could push this, I would start using right now. Thanks for the update.

Comment (rinaldilucas) [reply lib/rules/no-relative-imports.js]: I've applied what you suggested. Cheers from Brazil Daniil.

**Comment by qdanik** [comment]
New PR link: https://github.com/qdanik/eslint-plugin-path/pull/9

**Pull request closed without merging**